Description: Fixes the compatibility with the version of Aether/Maven Resolver in Debian
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/src/test/java/org/apache/maven/plugins/war/AbstractWarMojoTest.java
+++ b/src/test/java/org/apache/maven/plugins/war/AbstractWarMojoTest.java
@@ -37,7 +37,7 @@
 import org.codehaus.plexus.archiver.ArchiverException;
 import org.codehaus.plexus.archiver.jar.JarArchiver;
 import org.codehaus.plexus.util.FileUtils;
-import org.sonatype.aether.RepositorySystemSession;
+import org.eclipse.aether.RepositorySystemSession;
 
 public abstract class AbstractWarMojoTest
     extends AbstractMojoTestCase
@@ -332,4 +332,4 @@
         }
     }
 
-}
\ No newline at end of file
+}
